app/experimenter/experiments/tests/api/v5/test_queries.py

```diff
@@ -701,3 +701,118 @@ def test_paused_experiment_returns_date(self):
+            variables={"slug": experiment.slug},
+            headers={settings.OPENIDC_EMAIL_HEADER: user_email},
+        )
+        self.assertEqual(response.status_code, 200, response.content)
+        content = json.loads(response.content)
+        experiment_data = content["data"]["experimentBySlug"]
+        self.assertEqual(experiment_data["signoffRecommendations"]["qaSignoff"], True)
+        self.assertEqual(experiment_data["signoffRecommendations"]["vpSignoff"], True)
+        self.assertEqual(experiment_data["signoffRecommendations"]["legalSignoff"], False)
+
+    def test_risk_partner_related_signoff_recommendations(self):
+        user_email = "user@example.com"
+        experiment = NimbusExperimentFactory.create_with_status(
+            NimbusExperiment.Status.DRAFT, risk_partner_related=True
+        )
+
+        response = self.query(
+            """
+            query experimentBySlug($slug: String!) {
+                experimentBySlug(slug: $slug) {
+                    signoffRecommendations {
+                        qaSignoff
+                        vpSignoff
+                        legalSignoff
+                    }
+                }
+            }
+            """,
+            variables={"slug": experiment.slug},
+            headers={settings.OPENIDC_EMAIL_HEADER: user_email},
+        )
+        self.assertEqual(response.status_code, 200, response.content)
+        content = json.loads(response.content)
+        experiment_data = content["data"]["experimentBySlug"]
+        self.assertEqual(experiment_data["signoffRecommendations"]["qaSignoff"], True)
+        self.assertEqual(experiment_data["signoffRecommendations"]["vpSignoff"], True)
+        self.assertEqual(experiment_data["signoffRecommendations"]["legalSignoff"], True)
+
+    def test_risk_revenue_signoff_recommendations(self):
+        user_email = "user@example.com"
+        experiment = NimbusExperimentFactory.create_with_status(
+            NimbusExperiment.Status.DRAFT, risk_revenue=True
+        )
+
+        response = self.query(
+            """
+            query experimentBySlug($slug: String!) {
+                experimentBySlug(slug: $slug) {
+                    signoffRecommendations {
+                        qaSignoff
+                        vpSignoff
+                        legalSignoff
+                    }
+                }
+            }
+            """,
+            variables={"slug": experiment.slug},
+            headers={settings.OPENIDC_EMAIL_HEADER: user_email},
+        )
+        self.assertEqual(response.status_code, 200, response.content)
+        content = json.loads(response.content)
+        experiment_data = content["data"]["experimentBySlug"]
+        self.assertEqual(experiment_data["signoffRecommendations"]["qaSignoff"], True)
+        self.assertEqual(experiment_data["signoffRecommendations"]["vpSignoff"], True)
+        self.assertEqual(experiment_data["signoffRecommendations"]["legalSignoff"], True)
```

Comment:
1) I would do this combinatorial testing agaist the model method and then just a single test against the query to make sure it's hooked up

2) Might be able to compress this into a single parameterized test, something like

```py
@parameterized.expand([
    [True, False, True, False, True, False],
    .....
])
def test_testy_mctestface(self, risk_this, risk_that, risk_another, signoff_this, signoff_that, signoff_another):
    experiment = ExperimentFactory(risk_this=risk_this, etc....)
    self.assertEqual(experiment.signoff_recommendations, {
        'signoff_this': signoff_this',
        etc....
    })
```

Reply:
Hot damn I forgot about the parameterized thing. Thanks!